src/components/30-organisms/commercial-hero-banner/index.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { html, css, unsafeCSS } from 'lit-element';
+import '@axa-ch/container';
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+import childStyles from './child.scss';
+import InlineStyles from '../../../utils/inline-styles';
+
+class AXACommercialHeroBanner extends InlineStyles {
+  static get tagName() {
+    return 'axa-commercial-hero-banner';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      variant: { type: String },
+      imageSource: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.variant = 'light';
+    this.imageSource = '';
+  }
+
+  // Parent class InlineStyles needs a static method to retrive styles
+  // name of such method is passed when calling: this.inlineStyles('resetHeadingCss');
+  static get resetHeadingCss() {
+    return childStyles;
+  }
+
+  firstUpdated() {
+    this.inlineStyles('resetHeadingCss');
+    if (this.variant === 'dark') {
+      this.shadowRoot
```

Comment:
I would suggest to set classes in the render method. 

`const contentClasses = {
   'o-commercial-hero-banner__content': true,
   'o-commercial-hero-banner__text--dark': this.variant === 'dark',
};`

Reply:
I have to check. For some reason this is or was not possible if I remember correctly.